projections/weather/lib/weather/paginators.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# This is generated code!
+
+module Weather
+  # @api private
+  module Paginators
```

Comment:
Gonna need `.rbs` version of this file

Reply:
They are API private - I don't think we need one - no interaction will be done here, but we can probably add rbs for next_page etc.